package.json

```diff
@@ -54,13 +56,15 @@
     "onCommand:vscode-docker.browseDockerHub",
     "onCommand:vscode-docker.browseAzurePortal",
     "onCommand:vscode-docker.explorer.refresh",
+    "onCommand:vscode-docker.show-ACR-Task",
     "onCommand:vscode-docker.delete-ACR-Registry",
+    "onCommand:vscode-docker.run-ACR-Task",
     "onCommand:vscode-docker.delete-ACR-Repository",
     "onCommand:vscode-docker.delete-ACR-Image",
     "onCommand:vscode-docker.connectCustomRegistry",
     "onCommand:vscode-docker.setRegistryAsDefault",
     "onCommand:vscode-docker.disconnectCustomRegistry",
-    "onCommand:vscode-docker.pull-ACR-Image",
+    "onCommand:vscode-docker.pullFromAzure",
```

Comment:
Now that the -ACR- name is established, we might as well stick to it.

Reply:
yes, I've changed this as well in the next PR.